src/System.Data.Common/dir.props

```diff
@@ -5,7 +5,7 @@
     <AssemblyVersion>4.2.0.0</AssemblyVersion>
     <AssemblyKey>MSFT</AssemblyKey>
     <IsNETCoreApp>true</IsNETCoreApp>
-    <IsDesktopFacade>true</IsDesktopFacade>
     <IsUAP>true</IsUAP>
+    <IsNetFxNETStandard>true</IsNetFxNETStandard>
```

Comment:
No, it means that those are all the types/members we've agreed to be breaking changes on desktop.  Many of them are on types we can't OOB, and others (crypto) are support types for members on types we can't OOB.  There could be more work done in the libraries to try and make some of these oob-able but folks have already bought off on these breaking changes when they agreed to make netstandard2.0 supported on net461.  The ones I'm fixing are all the cases where it was broken because we were previously shipping an implementation/partial-facade and were broken when the standard repo only built pure-facades.